### ISSUE TITLE
rails plugin/engine ignores slim in gem dependancies

### ISSUE THREAD

**Issue by salex** [rails/rails#43933]
### Steps to reproduce

I've been playing with rails 7.x for a few months and got through the leaning process.

I decided to extract a few models out of an application and create a engine. Something I have not done since  Rails 3.x. I think the rails guide has not been updated since then but saw that it seemed to be upgraded.

So I follow the guide

`rails plugin new mybooks --mountable`

Fixed todo in gemspec to something sane.

Added `gem  "sprockets-rails"` to gem file after seeing the issue on it

added 'spec.add_dependency "slim-rails"' to gempec (and gemfile)

`bundle install` no problems, slim-rail in gemfile.lock

```ruby
bin/rails g scaffold book name root assets equity liabilities income expenses checking savings settings:text
```
It generated erb views

```

### System configuration

**Rails version**:
3.0.1
**Ruby version**:
7.0.0
```

I refuse to deal with erb - slim worked back in 3.x!  

Since this is a semi-refactor, I can deal with it or convert the erb files.

I then tried to see if I could get something to work in the dummy app.

I created some scaffolds and a model. Migrated  them.

Went to dummy and installed the migrations and migrated  them

No problem. I could go into the console and see the empty models.

I then tried to start the server and got

```sh
SteveAir:dummy $ bin/rails server
Could not find server ''.
Run `bin/rails server --help` for more options.
```
(that's two apostrophes not a quote.

Guess I'll have to dig a little more

**Comment by rafaelfranca** [comment]
Did you require the slim railtie after you moved the dependency to the engine? Only adding `slim-rails` to the gemspec doesn't require the railtie. 
Can you provide an example application that reproduce the issue?

**Comment by salex** [comment]
> Did you require the slim railtie after you moved the dependency to the engine? Only adding `slim-rails` to the gemspec doesn't require the railtie.
> Can you provide an example application that reproduce the issue?



The engine I tried to create [mybooksEngine](https://github.com:salex/mybooksEngine.git), was added my repository. It's just a bare shell with some scaffold/models.

I'm sorry for the delay in responding. I have no idea how to require the railtie for slim-rails. I tried searching for how you do that but couldn'd find anything. The slim-rails gem is in gemfile.lock but don't know how to make `rails g scaffold new model.. x x ` to generate slim templates.

Still can't figure out how to start the rails server in the dummy app. `rails s` just give the error `server '' can't be found`.
